cmd/root.go

```diff
@@ -21,6 +22,12 @@ var (
 	ErrFailedToValidateCredentials = errors.New("Failed to validate credentials")
 )
 
+// if non-zero, will log TLS keys to this file
+var UseTLSKeyLogFile = "yes"
+
+// SSL to be consistent with other producers, like Firefox and Chrome
+const TLSKeyLogFileEnv = "SSLKEYLOGFILE"
```

Comment:
I'm curious why we want this as the default? I thought we're checking for `if file != "" {` down below.

Reply:
We don't want it on by default. I just had it on by default in this draft. Have updated to be clearer